mysql-test/r/information_schema_ci.result

```diff
@@ -544,7 +544,9 @@ grant select (a) on test.t1 to joe@localhost with grant option;
 select * from INFORMATION_SCHEMA.COLUMN_PRIVILEGES WHERE table_schema != 'sys';
```

Comment:
For some reason permissions changed for this file `100644 → 100755`

Reply:
done